docs/docs/pods.md

```diff
@@ -0,0 +1,583 @@
+---
+title: Pods
+---
+
+# Pods
+
+Marathon supports the creation and management of pods. Pods enable you to share storage, networking, and other resources among a group of applications on a single agent and address them as one group rather than as separate applications, similar to a virtual machine. Pods allow quick, convenient coordination between applications that need to work together, for instance a database and web server that make up a content management system.
```

Comment:
I personally don't get the comparison to the virtual machine.   Also it is a terrible idea to have a db and a server in a pod IMO.. but that is an opinion.   Situations where pods make sense is the transition of legacy applications to a microservice based architecture (legacy being the important concept) and the addition of ancillary services to an application where there is a primary service and a analytics service or log scrapper is add to a pod with the primary service.